.github/workflows/evalml_main_performance_tests.yaml

```diff
@@ -0,0 +1,139 @@
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Run iterative algorithm performance test (current)
+        id: current_iterative
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo iterative --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.current_hash }} --job-desc ${{ steps.get_hashes.outputs.current_hash }}_iterative )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Run default algorithm performance test (previous)
+        id: previous_default
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo default --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.previous_hash }} --job-desc ${{ steps.get_hashes.outputs.previous_hash }}_default )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Run iterative algorithm performance test (previous)
+        id: previous_iterative
+        run: |
+          response=$( looking-glass run-evalml-automl --username machineFL --automl-algo iterative --datasets-yaml datasets_v2.yaml --evalml-branch ${{ steps.get_hashes.outputs.previous_hash }} --job-desc ${{ steps.get_hashes.outputs.previous_hash }}_iterative )
+          echo "::set-output name=job_id::$(echo $response | sed 's/.*Job ID: \([^ ]*\).*/\1/')"
+      - name: Await results
+        id: results
+        run: |
+          for id in ${{ steps.current_default.outputs.job_id }} ${{ steps.current_iterative.outputs.job_id }} ${{ steps.previous_default.outputs.job_id }} ${{ steps.previous_iterative.outputs.job_id }}; do
+            echo "Waiting for job id: $id"
+            result=''
+            sleep_time=0
+            while [ -z "$result"  ]
+            do
+              sleep $sleep_time
+              result=$(looking-glass get-job --job-id $id | grep -n 'COMPLETED' || :;)
+              sleep_time=60
+            done
+            looking-glass get-results --job-id $id --file-path $id
+          done
+        shell: bash
+      - name: Generate reports
+        run: |
+          filename_base=${{ steps.get_hashes.outputs.current_hash }}_v_${{ steps.get_hashes.outputs.previous_hash }}
+          looking-glass get-report --previous-path ${{ steps.previous_default.outputs.job_id }} -n ${{ steps.current_default.outputs.job_id }} --output-name ${filename_base}_default --output-type html
+          looking-glass get-report --previous-path ${{ steps.previous_iterative.outputs.job_id }} -n ${{ steps.current_iterative.outputs.job_id }} --output-name ${filename_base}_iterative --output-type html
```

Comment:
This will have to be updated once the[ looking glass server report generation ](https://github.com/alteryx/looking-glass/pull/546)gets merged in.

Reply:
Will it? Is the `get-report` command going away?

Reply:
Not really, but it might need to get renamed to `get-local-report`. Alternatively, there would also be a new command that kicks off the reporting job and stores it as a looking glass job. There's some more detail in the [updated user guide](https://github.com/alteryx/looking-glass/pull/546/files#) in that PR.

Reply:
can we just not change the name so it is backwards compatible?